sled-agent/src/bootstrap/client.rs

```diff
@@ -135,23 +108,14 @@ impl<'a> Client<'a> {
             .await
             .map_err(|err| Error::Connect { addr: self.addr, err })?;
 
-        let mut stream = crate::sp::maybe_wrap_stream(
-            stream,
-            self.sp,
-            SprocketsRole::Client,
-            Some(self.trust_quorum_members),
-            &self.log,
-        )
-        .await
-        .map_err(|err| Error::SprocketsSession(err.to_string()))?;
+        let mut stream = Box::new(tokio::io::BufStream::new(stream));
 
         // Build and serialize our request.
         let envelope = RequestEnvelope { version: version::V1, request };
 
         // "danger" note: `buf` contains a raw trust quorum share; we must not
```

Comment:
This comment can be removed too; there's no more "danger" to note.

Reply:
done.